### DISCUSSION TITLE
Switch Private to Public

### DISCUSSION THREAD

**Discussion by avenbreaks** [ConsenSys/quorum#1532] [General]
hello, lately I'm trying to make a private Blockchain using GoQuoum Dev Quickstart. Can I Switch a private Blockchain to public Blockchain and sync EVM?

**Comment by baptiste-b-pegasys** [comment]
Hello, what do you mean by private / public ?
You can disable private features (Tessera), you can let anyone accessing the API of your nodes, it depends.

**Reply by avenbreaks** [reply]
> Hello, what do you mean by private / public ? You can disable private features (Tessera), you can let anyone accessing the API of your nodes, it depends.

I mean can I build a public blockchain network like ethereum or polygon rename token chain id gas fee and production ready with goQuorum Dev Quickstart? I see GoQurom is only a private network

**Reply by baptiste-b-pegasys** [reply]
No I don't think so, quickdevstart is meant to quickly demonstrate the platform, for development.
Main reason is that the private keys are publicly available, so any example provided can not be used publicly, they are not secure.

QBS is production ready for example (managed service), and you can check the kubernetes repository: https://github.com/ConsenSys/quorum-kubernetes#production-network-guidelines

https://consensys.net/docs/goquorum/en/latest/deploy/deployment-overview/
